.gitattributes

```diff
@@ -0,0 +1 @@
+css/** linguist-generated
```

Comment:
Yeah I wouldn't prefer that. If I am guessing this right: you're adding this to avoid css changes showing up in the PR diffs?

Reply:
I actually don't know if this affects PR diffs (I usually don't track generated code or I put them in `dist/`), but I think it only affects [what github-linguist does](https://github.com/github/linguist/blob/master/docs/overrides.md). Sometimes it's useful to ignore generated code so that potential contributors can see the "real" primary language that they would be using (folders like `dist/` get ignored by default for this reason, otherwise a Typescript project might be detected as JavaScript).

Anyway, rebased away :slightly_smiling_face: